sqlmesh/integrations/github/cicd/controller.py

```diff
@@ -115,6 +115,35 @@ def is_skipped(self) -> bool:
         return self == GithubCheckConclusion.SKIPPED
 
 
+class BotCommand(Enum):
+    INVALID = 1
+    DEPLOY_PROD = 2
+
+    @classmethod
+    def from_comment_body(cls, body: str, namespace: t.Optional[str] = None) -> BotCommand:
+        body = body.strip()
+        command_aliases = {
+            cls.DEPLOY_PROD: {"/deploy_prod", "/deploy-prod", "/deployprod", "/deploy"},
+        }
+        if namespace:
+            command_aliases = {
+                command: {f"#{namespace}{alias}" for alias in alias_values}
```

Comment:
So it seems like `#` is not actually **required** and is indeed added automatically